lib/xeroizer/http_response.rb

```diff
@@ -133,7 +133,9 @@ def raise_error!
         when 400
           raise Xeroizer::BadResponse.new(description)
         when 401
-          raise OAuth::TokenExpired.new(description) if description.include?("TokenExpired")
+          if description && description.include?("TokenExpired")
```

Comment:
Is it better to use the safe operator? In instances where it is blank but a 401; it will silently fail.

Reply:
Yeah, silent failures are the worst. I'd rather get a `NoMethodError` in that case.

Reply:
it shouldn't fail if description is nil.